Framework/MDAlgorithms/src/ConvertMDHistoToMatrixWorkspace.cpp

```diff
@@ -222,7 +222,7 @@ void ConvertMDHistoToMatrixWorkspace::make1DWorkspace() {
     VMD wsCoord = start + dir * line.x[i];
 
     VMD inTargetCoord = transform->applyVMD(wsCoord);
-    outputWorkspace->dataX(0)[i] = inTargetCoord[id];
+    outputWorkspace->mutableX(0)[i] = inTargetCoord[id];
```

Comment:
If I do that I get the error `error: no match for ‘operator=’ (operand types are ‘Mantid::HistogramData::HistogramX’ and ‘Mantid::Kernel::VMD {aka Mantid::Kernel::VMDBase<float>}’)
   outputWorkspace->mutableX(0) = inTargetCoord;
`